### PR TITLE
Station flavour names replaced with strings lists!

### PR DESCRIPTION
**Pull request by praisenarsie**
Similar to #29622 except it's not lazy

I added a few other words to the various lists. The only one I removed is "Derp".

:cl: Frozenguy5
tweak: Added new station prefixes, names and suffixes!
/:cl:

### CODE REVIEW

File: strings/station_prefixes.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
```
Comment (Cyberboss) [inline strings/station_prefixes.txt]: Remove newline

File: strings/station_names.txt [hunk 2]
```diff
@@ -0,0 +1,85 @@
+
```
Comment (Cyberboss) [inline strings/station_names.txt]: Remove newline

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by AnturK** [comment]
Very nice, but i doubt English or Greek alphabet will sprout new letters in near future.

**Comment by Cyberboss** [comment]
Eh, it's really a code cleanliness thing at the cost of a small disk operation. I'm fine with it.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**